java/debugger.jpda/nbproject/project.properties

```diff
@@ -23,7 +23,7 @@ javac.source=1.8
 javadoc.arch=${basedir}/arch.xml
 jpda.classes.dir=${build.dir}/jpda/classes/
 requires.nb.javac=true
-spec.version.base=1.127.0
+spec.version.base=1.128.0
```

Comment:
Please remove the commit incrementing the version if targeting delivery - this will conflict with changes in master.

Reply:
Done. Good I have made that version increment as the last commit. Now it is gone.

Reply:
Done. Good I have made that version increment as the last commit. Now it is gone.